### PR TITLE
chore: [IA-532] Complete ZendeskSupportHelpCenter screen

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR complete the `ZendeskSupportHelpCenter` screen based on the UI provided.
It replicate the contextual help also for the Zendesk workflow.
The choice was to limit refactoring to a minimum trying to limit code reuse by extracting functions.

To maintain backward compatibility it was necessary to use navigation parameters.

This screen also manage the `PanicMode` case

## List of changes proposed in this pull request
- Complete `ZendeskSupportHelpCenter` screen
- Exported `getContextualHelpConfig` and `getContextualHelpData` functions and `reloadContextualHelpDataThreshold` const from `BaseScreenComponent` and `ContextualHelp` component
- Added navigation params to `zendeskSupportStart` action
- Added `ZendeskPanicMode` screen

https://user-images.githubusercontent.com/11773070/145780303-192f33ef-4ac9-4b85-998b-15c0aca021bc.mp4

https://user-images.githubusercontent.com/11773070/145780310-2cf4ee75-8ca4-45db-b5f7-8c7aaf04b58f.mp4

## How to test
Be sure to set the local `zendeskEnabled` FF to true and the remote `assistanceTool` to `zendesk` and press the assistance button (`?`)

### CODE REVIEW

File: locales/en/index.yml [hunk 1]
```diff
@@ -2900,6 +2900,13 @@ support:
     cta:
       contactSupport: "Contact support"
       seeReports: "See reports in progress"
+    supportComponent:
+      title: "Do you need help?"
```
Comment (thisisjp) [inline locales/en/index.yml:2904]:
```suggestion
      title: "Need help?"
```

File: locales/en/index.yml [hunk 2]
```diff
@@ -2900,6 +2900,13 @@ support:
...
+      subtitle: "To request assistance or suggest an improvement, write to us: we will reply as soon as possible."
```
Comment (thisisjp) [inline locales/en/index.yml:2905]:
```suggestion
      subtitle: "If you need support or want to suggest an improvement, please write to us: we will get back to you as soon as possible."
```

File: locales/en/index.yml [hunk 3]
```diff
@@ -2900,6 +2900,13 @@ support:
...
+      adviceMessage: "If your problem concerns the content of a message or the service of an organization, contact the latter. You will find the references within the message or in the card of the service in question."
```
Comment (thisisjp) [inline locales/en/index.yml:2906]:
```suggestion
      adviceMessage: "If your problem concerns the content of a message or the service of an Institution, contact the latter. You will find the contact details within the message or in the tab of the service concerned."
```

File: locales/en/index.yml [hunk 4]
```diff
@@ -2900,6 +2900,13 @@ support:
     cta:
       contactSupport: "Contact support"
```
Comment (thisisjp) [inline locales/en/index.yml:2901]:
```suggestion
      contactSupport: "Write us"
```

File: locales/en/index.yml [hunk 5]
```diff
@@ -2900,6 +2900,13 @@ support:
...
       seeReports: "See reports in progress"
```
Comment (thisisjp) [inline locales/en/index.yml:2902]:
```suggestion
      seeReports: "See your requests"
```

File: locales/en/index.yml [hunk 6]
```diff
@@ -2900,6 +2900,13 @@ support:
...
+    supportComponent:
+      title: "Do you need help?"
+      subtitle: "To request assistance or suggest an improvement, write to us: we will reply as soon as possible."
+      adviceMessage: "If your problem concerns the content of a message or the service of an organization, contact the latter. You will find the references within the message or in the card of the service in question."
+  panicMode:
+    title: "Unfortunately, we can't help you right now"
```
Comment (thisisjp) [inline locales/en/index.yml:2908]:
```suggestion
    title: "Sorry, we cannot help you at the moment"
```

File: locales/en/index.yml [hunk 7]
```diff
@@ -2900,6 +2900,13 @@ support:
...
+    body: "We know there is a problem and we are working to solve it. If the problem persists, or if you need assistance on another topic, please try again to write to us later."
```
Comment (thisisjp) [inline locales/en/index.yml:2909]:
```suggestion
    body: "We know there is a problem and we are working to solve it. If the problem remains, or if you need support on another topic, please try again later."
```

File: locales/it/index.yml [hunk 8]
```diff
@@ -2928,6 +2928,13 @@ support:
     cta:
       contactSupport: "Contatta l'assistenza"
```
Comment (thisisjp) [inline locales/it/index.yml:2929]:
```suggestion
      contactSupport: "Scrivici"
```

File: locales/it/index.yml [hunk 9]
```diff
@@ -2928,6 +2928,13 @@ support:
...
       seeReports: "Vedi segnalazioni in corso"
```
Comment (thisisjp) [inline locales/it/index.yml:2930]:
```suggestion
      seeReports: "Vedi le tue richieste"
```

**Review by thisisjp — COMMENTED** [review]
added new copy

**Review by thisisjp — APPROVED** [review]

**Review by fabriziofff — CHANGES_REQUESTED** [review]

File: ts/features/zendesk/screens/ZendeskPanicMode.tsx [hunk 10]
```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { SafeAreaView } from "react-native";
+import { useDispatch } from "react-redux";
+import image from "../../../../img/assistance/panicMode.png";
```
[the remark this answers is not in the document]
Comment (debiff) [reply ts/features/zendesk/screens/ZendeskPanicMode.tsx]: The second option! Thank you! Removed in [6f6767f](https://github.com/pagopa/io-app/pull/3571/commits/6f6767fa5d5fe18e775d5cb415aab911db982203)

**Review by fabriziofff — APPROVED** [review]

**Pull request merged**